### DISCUSSION TITLE
Incoming encoded query string array is not replaced properly

### DISCUSSION THREAD

**Discussion by mvenghaus** [livewire/livewire#8088] [7. Bugs (archived)]
### Livewire version

v3.4.8

### Laravel version

v10.47.0

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. create a property with Url attribute
2. use array in wire model
3. create a url in laravel and link to it

![livewire-url-replace](https://github.com/livewire/livewire/assets/6408081/4c74b3cd-722a-451b-afe4-b42a273f1532)

### Repository or Wirebox URL

https://github.com/mvenghaus/livewire-url-replace

### Additional information

When creating the link in laravel the array is url encoded. If i use the link from outside (in my case email) the incoming request is handled correctly. But if I change the property the browser url is not updated properly.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by gpluess** [comment]
I encountered the same problem today. Did you find a workaround in the meantime?
